blackbox/docs/sql/administration/ingestion_rules.txt

```diff
@@ -0,0 +1,105 @@
+.. highlight:: psql
+
+.. Hidden: CREATE TABLE::
+
+    cr> create table mqtt_temperature (
+    ... id long,
+    ... data string,
+    ... date timestamp
+    ... );
+    CREATE OK, 1 row affected (... sec)
+
+.. Hidden: CREATE TABLE::
+
+    cr> create table mqtt_humidity (
+    ... id long,
+    ... data string,
+    ... date timestamp
+    ... );
+    CREATE OK, 1 row affected (... sec)
+
+.. _administration-ingestion-rules:
+
+===============
+Ingestion Rules
+===============
+
+Ingestion rules define where and how the incoming data of a
+specific ingestion should be routed and stored.
+
+This document will show you how to:
+
+- Create ingestion rules
+- List existing ingestion rules
+- Delete ingestion rules
+
+.. SEEALSO::
+
+   :doc:`Learn more about ingestion <../../ingestion>`
+
+Creating Ingestion Rules
+========================
+
+Use the :ref:`create-ingest-rule` statement to create ingestion rules.
+
+Suppose we have an ingestion source for an ``mqtt`` broker, and we would like
+to filter temperature data streams by topic and write them to the
+``mqtt_temperature`` table.
+
+We could do this::
+
+    cr> CREATE INGEST RULE mqtt_temperature_rule
+    ... ON mqtt
+    ... WHERE topic like 'temperature/%'
```

Comment:
I still think that we should explain that the topic here is specific to the `mqtt` ingestion source and that the available fields in the where clause are ingestion source specific.

We also still don't explain that the schema of the data and how it's mapped to the target table is also specific to the ingestion source.

From my experience seeing examples that contain unexplained stuff can be very frustrating when attempting to learn something.